website/views.py

```diff
@@ -366,3 +366,7 @@ def redirect_about(**kwargs):
 
 def redirect_howosfworks(**kwargs):
     return redirect('/getting-started/')
+
+
+def redirect_meetings_analytics_link(**kwargs):
+    return redirect('/getting-started/?utm_source=osf4m&utm_medium=email&utm_campaign=success')
```

Comment:
I'll assume for the time being that we don't actually want the `#addons` fragment in the first case, which is the one I'm seeing here. So this is fine. :)